src/test/java/co/rsk/federate/signing/hsm/advanceblockchain/HsmBookkeepingClientImplTest.java

```diff
@@ -305,14 +306,17 @@ public void advanceBlockchain_with_brothers_ok() throws HSMClientException, Json
         assertEquals(Hex.toHexString(blocks.get(1).getHeader().getFullEncoded()), blocksInRequest.get(1).asText());
         assertEquals(Hex.toHexString(blocks.get(0).getHeader().getFullEncoded()), blocksInRequest.get(2).asText());
 
-        // Assert block brothers
-        assertEquals(blocks.get(0).getUncleList().size(), brothersInRequest.get(2).size());
-        assertEquals(blocks.get(1).getUncleList().size(), brothersInRequest.get(1).size());
-        assertEquals(blocks.get(2).getUncleList().size(), brothersInRequest.get(0).size());
+        // Assert brothers
+        List<BlockHeader> block2Uncles = blocks.get(1).getUncleList();
+        String block1Brothers = brothersInRequest.get(2).toString();
+        assertTrue(block1Brothers.contains(Hex.toHexString(block2Uncles.get(0).getFullEncoded())));
```

Comment:
Shouldn't it be an equals instead of contains?

Reply:
The Brothers Are In A JsonText Containing Square Bracket And Quotes, That Will Require Formatting To Separate